core/src/main/java/org/frankframework/monitoring/MonitorManager.java

```diff
@@ -126,17 +127,12 @@ public Monitor getMonitor(int index) {
 		return monitors.get(index);
 	}
 	public Monitor findMonitor(String name) {
```

Comment:
Kan dit niet gelijk `public Optional<Monitor> findMonitor(String name)` worden? Zo te zien wordt dit alleen in `org.frankframework.management.bus.endpoints.Monitoring#getMonitor` gebruikt, en die is er ook makkelijk op aan te passen.